### PR TITLE
Cleanups for #1763

### PR DESCRIPTION
**Pull request by The-Compiler**
Consider this my after-the-fact review for #1763 :wink:

cc @tomviner @JonathonSonesen

### CODE REVIEW

**Comment by RonnyPfannschmidt** [comment]
looks good to me, since the ci didnt run on it please merge after a local test

File: _pytest/main.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def wrap_session(config, doit):
                     excinfo.value, pytest.exit.Exception):
                 excinfo = _pytest._code.ExceptionInfo()
```
Comment (RonnyPfannschmidt) [inline _pytest/main.py]: isnt this one double ?

Comment (The-Compiler) [inline _pytest/main.py]: Hmm, yeah, probably... Maybe @tomviner can shed some light on it?

Comment (tomviner) [inline _pytest/main.py]: Yes, my bad, that's a left from over from a different approach. Please delete that line.

**Comment by tomviner** [comment]
@The-Compiler thanks for the cleanup here. Much appreciated.

**Comment by The-Compiler** [comment]
The double line is gone now.

**Comment by tomviner** [comment]
:+1:

**Pull request merged**